### DISCUSSION TITLE
Server not functioning- "Did not connect: Potential Security Issue

### DISCUSSION THREAD

**Discussion by FalcoSkyWolf** [mastodon/mastodon#25747] [Q&A] [answered]
My server (mastodon.sdf.org) was working fine until I restarted my computer. Now it's giving me an error message.

Did Not Connect: Potential Security Issue

Firefox detected an issue and did not continue to mastodon.sdf.org. The website is either misconfigured or your computer clock is set to the wrong time.

It’s likely the website’s certificate is expired, which prevents Firefox from connecting securely.

What can you do about it?

mastodon.sdf.org has a security policy called HTTP Strict Transport Security (HSTS), which means that Firefox can only connect to it securely. You can’t add an exception to visit this site.

The issue is most likely with the website, and there is nothing you can do to resolve it. You can notify the website’s administrator about the problem.

I would really rather not start another account in another instance if I can avoid it- I might be a new user but I've already been fairly busy and had gotten comfy.

**Comment by StrangeMinds1960** [comment, marked as answer]
Site works fine.  A new SSL certificate was issued today so it's likely there was a problem with automatic updates (it's a LetsEncrypt certificate) and it had to be done manually

**Reply by FalcoSkyWolf** [reply]
Thanks. Do you know if this is a common occurrence and if it usually gets fixed this quickly? I can handle a BIT of downtime but need access to it daily at least.

**Reply by StrangeMinds1960** [reply]
No idea I'm afraid, your best bet is to get hold of the admin(s) of that server to find out what went wrong.  LetsEncrypt can be finnicky (I know from experience) so maybe they just didn't set auto-renew up properly
